shared/src/test/diff/codegen/Polyfill.mls

```diff
@@ -41,15 +41,18 @@ n = 123 with { x = 1 }
 //│   return !x;
 //│ }
 //│ let res;
-//│ // Query 0
-//│ const n = withConstruct(123, {  x: 1  }); res = n;
+//│ // Query 1
+//│ const n = withConstruct(123, { 
+//│ x: 1
```

Comment:
Why the strange indentation?

Reply:
I’m investigating it. 🧐

Reply:
Fixed! https://github.com/hkust-taco/mlscript/blob/80d13bbcc89cf501557aeb152c1ada3c8e93cd93/shared/src/test/diff/codegen/Polyfill.mls#L47